src/main/java/com/cloudbees/groovy/cps/tool/Driver.java

```diff
@@ -72,24 +73,28 @@ public void run() throws Exception {
 
             final DeclaredType closureType = t.types.getDeclaredType(t.elements.getTypeElement(Closure.class.getName()));
 
-            /**
-             * Criteria:
-             *      1. public static method
-             *      2. has a Closure parameter in one of the arguments, not in the receiver
-             */
             Predicate<ExecutableElement> selector = (e) -> {
-                return e.getKind() == ElementKind.METHOD
-                    && e.getModifiers().containsAll(PUBLIC_STATIC)
-                    && e.getParameters().subList(1, e.getParameters().size()).stream()
-                        .anyMatch(p -> t.types.isAssignable(p.asType(), closureType))
-                    && !EXCLUSIONS.contains(e.getEnclosingElement().getSimpleName().toString()+"."+e.getSimpleName().toString());
+                boolean r =
+                        // Only interested here in methods; not currently handling nested classes.
+                        e.getKind() == ElementKind.METHOD
+                        // Top-level invocations can only be to public static methods. But some private/protected static helper methods need translation, too.
+                        && e.getModifiers().contains(Modifier.STATIC)
+                        // Has a Closure parameter in one of the arguments (not in the receiver).
+                        && e.getParameters().subList(1, e.getParameters().size()).stream()
```

Comment:
Looks like this needs to be `sublist(0, ...)` - `callClosureForMapEntry` and the like have `Closure` as the first parameter.